### ISSUE TITLE
Import k3 cluster into Rancher

### ISSUE THREAD

**Issue by goffinf** [rancher/k3s#128]
Thanks for helping us to improve k3s! We welcome all bug reports. At this stage, we are also looking for help in testing/QAing fixes. Once we've fixed you're issue, we'll ping you in the comments to see if you can verify the fix. We'll give you details on what version can be used to test the fix. Additionally, if you are interested in testing fixes that you ***didn't*** report, look for the issues with the `status/to-test` label. You can pick any of these up for verification.  ***You can delete this message portion of the bug report.***

**Describe the bug**
Possibly related to .. https://github.com/rancher/k3s/issues/69

Created a 3 node k3s cluster using docker-compose - successful

Performed a single node install of Rancher v2.1.6 (using self-signed cert option) - successful

Attempt to import k3s cluster - FAILED

```
This cluster is currently Pending; areas that interact directly with it will not be available until the API is ready.

Waiting for full cluster configuration
```

```
kubectl -n cattle-system get pod
NAME                                   READY   STATUS             RESTARTS   AGE
cattle-cluster-agent-9564bc646-bxj6s   0/1     CrashLoopBackOff   4          4m20s
cattle-node-agent-bgs5v                0/1     CrashLoopBackOff   4          4m20s
cattle-node-agent-sz52c                0/1     CrashLoopBackOff   4          4m20s
cattle-node-agent-wmqcw                0/1     CrashLoopBackOff   4          4m20s

kubectl -n cattle-system logs cattle-cluster-agent-9564bc646-bxj6s
INFO: Environment: CATTLE_ADDRESS=10.42.5.4 CATTLE_CA_CHECKSUM=5f8e57ca7c3a750c43c723e5b9219aa3658d7b8bc71938bdcbf60e234384919e CATTLE_CLUSTER=true CATTLE_INTERNAL_ADDRESS= CATTLE_K8S_MANAGED=true CATTLE_NODE_NAME=cattle-cluster-agent-9564bc646-bxj6s CATTLE_SERVER=https://localhost
INFO: Using resolv.conf: search cattle-system.svc.cluster.local svc.cluster.local cluster.local Home nameserver 10.43.0.10 options ndots:5
ERROR: https://localhost/ping is not accessible (Failed to connect to localhost port 443: Connection refused)

kubectl -n cattle-system logs cattle-node-agent-bgs5v
ERROR: Please bind mount in the docker socket to /var/run/docker.sock
ERROR: example: docker run -v /var/run/docker.sock:/var/run/docker.sock ...
```

**To Reproduce**

1.  Create a 3 node k3s cluster using docker-compose
2.  Perform a single node install of Rancher v2.1.6 (using self-signed cert option)
3.  Attempt to import k3s cluster

**Expected behavior**

Successful import of K3s cluster into Rancher.

**Screenshots**

**Additional context**

Possibly related to ... https://github.com/rancher/k3s/issues/69

**Comment by ibuildthecloud** [comment]
This works in on master of rancher (so `rancher/rancher:master`) and will be included in Rancher 2.2.  Fixed in https://github.com/rancher/rancher/pull/18461

**Comment by ibuildthecloud** [comment]
As noted, this is the being addressed in #69, I'm going to close.  You can track #69
